src/cli/extensions/error.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @ignore
```

Comment:
For some reason this ends up being 7 undocumented functions, felt that it incorrectly represented ESDocs coverage. We want documentation here as in general.